admin/client/src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import SilverStripeComponent from 'lib/SilverStripeComponent';
+import { Tab, Nav, NavItem } from 'react-bootstrap-ss';
+
+class Tabs extends SilverStripeComponent {
+  /**
+   * Determines a default tab to be opened and validates the given default tab.
```

Comment:
Tabs can have children, which can be (nested) Tabs as well, right? This only iterates through the top level